### PR TITLE
CSS143B-2020Fall-Homework_1

### PR DESCRIPTION
**Pull request by malasidbaerot**
Homework1 deadline 10/8 /2020 11:59 pm

### CODE REVIEW

File: src/test/java/Problem2Test.java [hunk 1]
```diff
@@ -1,14 +1,22 @@
...
+//importing assert library
 public class Problem2Test {
-
-    // Example test
-    // Okay to change
     @Test
     public void testBubbleSort() {
-        assertTrue(false);
-        // add your tests here
+        int[] [] data= {{1,3,2,4,5},{1,3,2},{2,1},{1}};
```
Comment (pdgetrf) [inline src/test/java/Problem2Test.java:8]: good test!

consider add a few more for the edge cases such as empty array, null array, etc.

File: src/test/java/Problem2Test.java [hunk 2]
```diff
@@ -1,14 +1,22 @@
 import org.junit.Test;
-
+//importing junit test
 import static org.junit.Assert.*;
-
+//importing assert library
```
Comment (pdgetrf) [inline src/test/java/Problem2Test.java:4]: remove code not used for submission.

**Review by pdgetrf — APPROVED** [review]
LGTM
